lamindb/_record.py

```diff
@@ -224,14 +229,25 @@ def __init__(record: Record, *args, **kwargs):
                 message = _format_django_validation_error(record, e)
                 raise FieldValidationError(message) from e
     elif len(args) != len(record._meta.concrete_fields):
-        raise ValueError("please provide keyword arguments, not plain arguments")
+        sys.exit(
+            f"Use keyword arguments instead of positional arguments, e.g.: {record.__class__.__name__}(name='...'). See model documentation for valid fields."
+        )
     else:
         # object is loaded from DB (**kwargs could be omitted below, I believe)
         super(BasicRecord, record).__init__(*args, **kwargs)
         _store_record_old_name(record)
         _store_record_old_key(record)
 
 
+def __init__(record: Record, *args, **kwargs):
+    try:
+        _init_with_potential_errors(record, *args, **kwargs)
+    except ValueError as e:
+        if "missing required argument" in str(e):
+            sys.exit(str(e))
```

Comment:
raise SystemExit or one of the custom exceptions

Reply:
Wait, no. Instead of doing this we should update the ValueError that is raised this in the first place with a `FieldValidationError` (and move this into the exceptions: https://github.com/laminlabs/lamindb/blob/bf44bbf579ca403a1c40735f4d3174ed58240dac/lamindb/base/validation.py#L9; it's currently here because of lnschema-core migration).

I see that according to our code it should be a `TypeError` anyway. But yeah, please simply replace with FieldValidationError and then the __init__ can stay as it is: https://github.com/laminlabs/lamindb/blob/bf44bbf579ca403a1c40735f4d3174ed58240dac/lamindb/_record.py#L108

Reply:
I'm not sure whether I got what you're saying here correctly despite me reading it a couple of times.

1. I moved the `FieldValidationError` to `exceptions.py`.
2. We are now raising a `FieldValidationError` instead of a `ValueError`.
3. The `__init__` now stays as it was.

This gives us this experience:

```
An exception has occurred, use %tb to see the full traceback.

FieldValidationError: Use keyword arguments instead of positional arguments, e.g.: Organism(name='...'). See model documentation for valid fields.
```

(for `bt.Organism("bla")`)

Reply:
This reads correct. I'm looking at the code now.